source/config/featureFlag.py

```diff
@@ -0,0 +1,141 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+"""Manages NVDA configuration.
+Provides utility classes to make handling featureFlags easier.
+"""
+
+import enum
+from typing import (
+	Optional,
+)
+from configobj.validate import (
+	ValidateError,
+	VdtParamError,
+)
+from logHandler import log
+
+
+class FeatureFlagValue(enum.Enum):
+	""" The explicit DEFAULT option allows developers to differentiate between a value set that happens to be
+	the current default, and a value that has been returned to the "default" explicitly.
+	"""
+	DEFAULT = enum.auto()
+	DISABLED = enum.auto()
+	ENABLED = enum.auto()
+
+
+class FeatureFlag:

```

Comment:
Even if not handled in this PR, it would be interesting to indicate how you imagine to handle non-boolean feature choices in the future, e.g. advanced option for UIA in Word documents which has 3 choices plus the default.
Would it be worth to reserve the name `FeatureFlag` for generic flag that may have any number of values (choices) and rename the `FeatureFlag` class to something like `BooleanFeatureFlag`? Or even better, implement `FeatureFlag` with all the generic stuff and implement a subclass `BooleanFeatureFlag` containing all what is specifically related to a 2-value choice (excepted Default)?
A `MultiChoiceFeatureFlag` class could be implemented subsequently.

Reply:
Please take a look at https://github.com/nvaccess/nvda/pull/13871